test/hotspot/jtreg/runtime/CommandLine/PrintClasses.java

```diff
@@ -30,6 +30,14 @@
  * @run main/othervm PrintClasses
  */
 
+/*
+ * @test
+ * @bug 8298162
+ * @summary Test jcmd VM.classes
+ * @library /test/lib
+ * @run main/othervm -XX:StartFlightRecording PrintClasses
```

Comment:
I think if you want to run JFR you also need:
```
 * @requires vm.hasJFR
 ```
and possibly:
```
* @key jfr
```

Reply:
Thanks David. I copied that line from another test then forgot to put it in mine.  I don't see any other @key jfr tests in the runtime directory so I'm not sure I need that.

Reply:
Only the actual jdk/jdk/jfr tests have @key jfr.

Reply:
Right - that is where I saw it. I don't know the significance of it.

Reply:
The "@key jfr" was originally a means to disallow additional parameters from being passed onto a test from the outside, externally. We have marked our JFR tests with this key to disallow external parameters because most tests are sensitive.

I think in this case you would not need "@key jfr".